Bezusov_1-2LessonHW.html

```diff
@@ -0,0 +1,181 @@
+        var y = 0.7;
+        var sum = x + y;
+        document.write(sum.toFixed(2) + '<br>') ;
+// 5)
+        var price = 100;
+        var strPrice = price.toString();
+        document.write(strPrice + '$' + '<br>');
+
+// Lesson 2 home work (string's)
+
+//// 1) object 
+        var phone = {
+        product: 'iphone',
+        };
+
+        phone.price = 1000;
+        phone.details = {
+                color:'red',
+                model: '6s'
+        };
+///// 2)
+
+        //let s = 0||'string'; /* Тут равно - string так как первый елемент Fasle */
+        //let s = 1 && 'string'; /* Равно string так как первый елемент Тру и оно смотрит дальше*/
+        //let s = null || 25; /* Равно 25 так как первое значение False и смотрит дальше*/
+        //let s = null && 25; /* так как первое значение False оно падает*/
+        //let s = null || 0 || 35; /* так как единственное значение true - 35 выводит его*/
+        //let a = null && 0 && 35; /* так как первое значение False оно падает*/
+
+// 3)
+
+        // 12 + 14 + '12' /* =2612 так как первые два числа щитаються и плюсуется стринга в конце */
+        // 3 + 2 - '1' /* =4 так как один минус (математическая операцыя) это не конкатинацыя оно превращает стрингу в номер*/
+        // '3' + 2 - 1 /* 31 тринга плюс 1*/
+        // true + 2 /* =3 так как тру булинг = 1*/
+        // +'10' + 1 /* =11 так как у тринги есть дополнительный плюс превращает в номер */
+        //undefined + 2 null + 5 /* ошыбка*/
+        // true + undefined /* =NAN true + undefined */
+
+// if else (задачи)
+        // 1)
+                // var block = 'hidden';
+                // if(block = 'hidden'){
+                //         block = 'visible';
+                // }else{
+                //         block = 'hidden';
+                // }
+        // 2)
+                // var zero = 1;
+                // if(zero == 0){
+                //         zero = 1;
+                // } else if (zero <= 0){
+                //         zero = 'less then zerro';
+                // } else if (zero >= 0){
+                //         zero = zero * 10;
+                // }
+        // 3)
+                // let car = { 
+                //         name: 'Lexus',
+                //         age: 10,
+                //         create: 2008,
+                //         needRepair: false
+                // }
+                // if (car.age >= 5){
+                //         alert('Need repair');
+                //         car.needRepair = true
+                // } else{
+                //         car.needRepair = false
+                // }
+        // 4)
+                // let item = { name: 'Intel core i7',
+                //         price: '100$',
+                //         discount: '15%' 
+                // }
+                // if (item.discount){
+                //         item.priceWithDiscount = parseInt(item.price) - (parseInt(item.discount) * parseInt(item.price) / 100) + '$' ;
```

Comment:
Better keep `parseInt(item.price)` in separate variable if it's used more then once